compressors/compressor.go

```diff
@@ -0,0 +1,8 @@
+package compressors
+
+// Compressor declares the functionality provided by the different compression logics supported.
+type Compressor interface {
+	GetType() string
```

Comment:
can you remind me why do we need to GetType function?
how is it used? can we do without it?
would be good if you have a pointer to send me where I can see usage example

Reply:
@vMaroon where do we use GetType?